### PR TITLE
pricing primary region should be us-east

### PR DESCRIPTION
**Pull request by kikiya**

### CODE REVIEW

File: solutions/02-follower-reads/movr/pricing/data/pricing_database.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
 CREATE DATABASE movr_pricing;
 
-ALTER DATABASE movr_pricing SET PRIMARY REGION "us-west";
+ALTER DATABASE movr_pricing SET PRIMARY REGION "us-east";
 ALTER DATABASE movr_pricing ADD REGION "us-central";
-ALTER DATABASE movr_pricing ADD REGION "us-east";
+ALTER DATABASE movr_pricing ADD REGION "us-west";
```
Comment (dweatherbee) [inline solutions/02-follower-reads/movr/pricing/data/pricing_database.sql:6]: I know this is default but to keep consistent we need to add:

ALTER DATABASE movr_pricing SURVIVE REGION FAILURE;

Comment (dweatherbee) [reply solutions/02-follower-reads/movr/pricing/data/pricing_database.sql:6]: The solution 4 file also needs to have this added.

**Review by dweatherbee — CHANGES_REQUESTED** [review]

File: solutions/02-follower-reads/movr/pricing/data/pricing_database.sql (hunk 1 above)
Comment (kikiya) [reply solutions/02-follower-reads/movr/pricing/data/pricing_database.sql:6]: added to those solutions!

**Review by dweatherbee — APPROVED** [review]
Thanks looks great!

**Pull request merged**